webrtc.html

```diff
@@ -12001,8 +12001,8 @@ <h3 id="sec-constraints">Constraints</h3>
       default values for attributes that would be left uninitialized without
       the init dictionary present.</li>
       <li>InvalidMediaStreamTrackError changed to InvalidParameter.</li>
-      <li>Fire NetworkError when the data transport is closed with an
-      error.</li>
+      <li>Fire the <code>RTCDataChannel.onerror</code> EventHandler
```

Comment:
I think you should "fire an event named Error at the DataChannel object, with "message" set to an appropriate error description".
If we need more data than "an error happened, write this to the debug log", we need to use a specialization of RTCError.